src/sql/src/plan/statement/ddl.rs

```diff
@@ -2457,6 +2459,10 @@ fn kafka_sink_builder(
                 sql_bail!("Must specify both AVRO KEY FULLNAME and AVRO VALUE FULLNAME when specifying generated schema names");
             }
 
+            // If NULL DEFAULTS was specified without TRUE | FALSE, it should be equivalent to NULL DEFAULTS TRUE
+            let null_defaults =
```

Comment:
This is already the default behavior of `bool` config options. https://github.com/MaterializeInc/materialize/blob/57bff0c69ec5212e9524b9dfd04d58edb9ef4973/src/sql/src/plan/with_options.rs#L230-L234

Reply:
Ah I was not seeing that behaviour. Let me update with the `Default(bool)`.

Reply:
done!